content/kotsadm/installing/installing-embedded-cluster.md

```diff
@@ -32,11 +32,20 @@ tar xvf supergoodtool.tar
 cat install.sh | sudo bash -s airgap
 ```
 
-Note that the airgapped installer is not the same as an [airgapped application package](/kotsadm/installing/airgap-packages/).
-A KOTS application may be installed in airgapped mode on clusters installed online and vice versa.
-
 kURL currently uses `.tar.gz` extension for a `.tar` file, hence the `-o *.tar`.
 
+Note that the airgapped installer in the above step is not the application installer.  After this command completes, the application can be installed using the application airgap bundle.
```

Comment:
This can be clearer. If it's not the application installer, what is it? Does it have a name?

Reply:
Maybe we need a meta section that clarifies that installing an app to an embedded cluster is a two-step process. First create the cluster with Admin Console included, then start Admin Console and install the application.

Reply:
Going further, Clarifying that there are two things to download might be important, given this is airgapped, and you need to download them both (plus the license) and *then* run the commands.